src/Parser.java

```diff
@@ -0,0 +1,11 @@
+package src;
+
+import java.io.IOException;
+
+/**
+ * Created by Антон on 04.08.2016.
+ */
+public interface Parser {
```

Comment:
Something is wrong with the name, it says `Parser` but does parse nothing. It just reads/saves..